### PR TITLE
Remove deprecated `componentWillMount`

### PR DESCRIPTION
**Pull request by YuCJ**
KanbanFlow task: https://kanbanflow.com/t/2dYaDAaH

### CODE REVIEW

**Comment by YuCJ** [comment]
TODO: Release production after testing

File: packages/universal-header/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "querystring": "^0.2.0",
     "react": "^16.3.0",
     "react-redux": "^6.0.0",
-    "react-router-dom": "^4.3.1",
```
Comment (nickhsine) [inline packages/universal-header/package.json]: Why we upgrade `react-router-dom` from v4 to v5?
Our main repo(twreporter-react) still requires v4, won't they conflict?

Comment (YuCJ) [reply packages/universal-header/package.json]: `componentWillMount` and `componentWillReceiveProps` are removed in v5.0.0
https://github.com/ReactTraining/react-router/releases

I'll update the dependency in `twreporter-react` after this package published.

**Comment by nickhsine** [comment]
LGTM

**Pull request merged**